src/commcare_cloud/fab/utils.py

```diff
@@ -153,15 +153,14 @@ def diff(self):
         return DeployDiff(self.repo, self.last_commit_sha, self.deploy_ref)
 
 
-GITHUB_CREDENTIALS = None
+GITHUB_TOKEN = None
 
 
-def _get_github_credentials(message, force=False):
-    global GITHUB_CREDENTIALS
+def _get_github_token(message, force=False):
+    global GITHUB_TOKEN
 
-    if GITHUB_CREDENTIALS is not None:
-        if not force or GITHUB_CREDENTIALS[0]:
-            return GITHUB_CREDENTIALS
+    if GITHUB_TOKEN is not None and not force:
```

Comment:
I think this logic is wrong. The intention is to allow blank tokens unless 'force=true' in which case a token is required. I've also dropped the `not None` since an empty token is equivalent to `None` as far as the Github API library is concerned.

```suggestion
    if GITHUB_TOKEN or not force:
```

Reply:
Thanks I didn't understand the intention originally but it makes sense now. Made the change and updated the parameter name to from `force` to `required`.